### PR TITLE
Feature/436 Added Saturation to User and Appointment

### PR DESCRIPTION
**Pull request by Narendran2020**

### CODE REVIEW

File: LIMS.Business/Injectors/AppointmentInjector.cs [hunk 1]
```diff
@@ -17,10 +18,12 @@ public sealed class AppointmentInjector : IInjector<Appointments>
             ILimsRepository<User> userRepository,
             ILimsRepository<Department> departmentRepository,
             ILimsRepository<AppointmentStatus> appointmentStatusRepository,
-            ILimsRepository<ClinicianSchedule> clinicianScheduleRepository)
+            ILimsRepository<ClinicianSchedule> clinicianScheduleRepository,
+            ILimsRepository<Salutation> salutationRepository)
```
Comment (Aswathyshibu) [inline LIMS.Business/Injectors/AppointmentInjector.cs:22]: Please fix the alignment

File: LIMS.Business/Injectors/UserInjector.cs [hunk 2]
```diff
@@ -16,11 +17,13 @@ public class UserInjector : IInjector<User>
         public UserInjector(
             ILimsRepository<Role> roleRepository,
             ILimsRepository<Department> departmentRepository,
-            ILimsRepository<Qualification> qualificationRepository)
+            ILimsRepository<Qualification> qualificationRepository,
+            ILimsRepository<Salutation> salutationRepository)
```
Comment (Aswathyshibu) [inline LIMS.Business/Injectors/UserInjector.cs:21]: Fix the alignment

File: LIMS.Business/Injectors/UserInjector.cs [hunk 3]
```diff
@@ -16,11 +17,13 @@ public class UserInjector : IInjector<User>
...
         {
             this._roleRepository = roleRepository;
             this._departmentRepository = departmentRepository;
             this._qualificationRepository = qualificationRepository;
+            this._salutationRepository = salutationRepository;
```
Comment (Aswathyshibu) [inline LIMS.Business/Injectors/UserInjector.cs:26]: alignment

**Review by Aswathyshibu — CHANGES_REQUESTED** [review]

File: LIMS.Business/Injectors/UserInjector.cs (hunk 3 above)
Comment (Narendran2020) [reply LIMS.Business/Injectors/UserInjector.cs:26]: done

File: LIMS.Business/Injectors/UserInjector.cs (hunk 2 above)
Comment (Narendran2020) [reply LIMS.Business/Injectors/UserInjector.cs:21]: Done

File: LIMS.Business/Injectors/AppointmentInjector.cs (hunk 1 above)
Comment (Narendran2020) [reply LIMS.Business/Injectors/AppointmentInjector.cs:22]: Done

**Review by arunselvakumar — APPROVED** [review]

**Comment by arunselvakumar** [comment]
Looks good, Approved!

**Review by Aswathyshibu — APPROVED** [review]

**Pull request merged**